lib/runCall.js

```diff
@@ -128,6 +128,26 @@ module.exports = function (opts, cb) {
     ethereumConfig.setEip150blocknumber(new Uint8Array([0]))
     ethereumConfig.setEip160blocknumber(new Uint8Array([0]))
     ethereumConfig.setEip161blocknumber(new Uint8Array([0]))
+    var hardfork = stateManager._common._hardfork
+    switch (hardfork) {
+      case 'byzantium': {
+        ethereumConfig.setByzantiumblocknumber(new Uint8Array([0]))
+        ethereumConfig.setConstantinopleblocknumber(new Uint8Array([108, 8, 65])) // 0x6C0841
+        ethereumConfig.setPetersburgblocknumber(new Uint8Array([111, 21, 129]))   // 0x6F1581
```

Comment:
Why these numbers in particular?

Reply:
No particular reason. I chose them because these are the block numbers at which the fork has been updated.

Reply:
Let's pick bigger numbers. If for whatever reason we end up with "real" block numbers, this will behave irregularly. Why don't you just go with [127, 255, 255, 255] for both of them.

Reply:
Sounds good! I will make all the changes